inject/chrome-mv3.js

```diff
@@ -0,0 +1,36 @@
+/* global contentScopeFeatures */
+
+const secret = window.crypto.randomUUID()
+
+contentScopeFeatures.load()
+
+window.addEventListener(secret, ({ detail: message }) => {
+    switch (message?.type) {
+    case 'update':
+        contentScopeFeatures.update(message)
+        break
+    case 'register':
+        // Background has disabled features
+        if (!message?.argumentsObject) {
+            return
+        }
+
+        if (message?.argumentsObject.debug) {
+            window.addEventListener('message', (m) => {
+                if (m.data.action && m.data.message) {
+                    chrome.runtime.sendMessage({
```

Comment:
This won't work will it?
We're in the page world now so we'd have to pass this back to the isolated world first.

Reply:
Doh, good point. Done.